webdriver/tests/support/fixtures_bidi.py

```diff
@@ -1,15 +1,22 @@
 import asyncio
-from typing import Any, Mapping
+from typing import Any, Callable, Mapping, Optional
 
 import pytest
 import webdriver
 from webdriver.bidi.modules.script import ContextTarget
 
+CONTEXT_LOAD_EVENT = "browsingContext.load"
 
 @pytest.fixture
-async def new_tab(bidi_session):
+async def new_tab(bidi_session, wait_for_event):
     """Open and focus a new tab to run the test in a foreground tab."""
+
+    # Subscribe to the load event to wait for the newly created tab to be loaded.
+    await bidi_session.session.subscribe(events=[CONTEXT_LOAD_EVENT])
     new_tab = await bidi_session.browsing_context.create(type_hint='tab')
+    await wait_for_event(CONTEXT_LOAD_EVENT,
+                         lambda data: data["context"] == new_tab["context"])
+    await bidi_session.session.unsubscribe(events=[CONTEXT_LOAD_EVENT])
```

Comment:
FWIW adding `session.subscribe`/`unsubscribe` commands in helpers/fixtures might have unexpected side effects.

If for any reason there was already a call to `subscribe(events=[CONTEXT_LOAD_EVENT])` before we run this fixture, the new `subscribe` command will have no effect, but the `unsubscribe` will fully unsubscribe from the event. In that case we are in a fixture, which should always run at the beginning of a test, so it is safe but still concerning.

Reply:
Valid point, but IDK how to fix the issue in another way. Introducing a global subscription state looks odd. Do you have any ideas?

Reply:
If we really can't make `browsingContext.create` to wait until the browsing context is loaded, I don't have any good solution either. Not sure what's better between using subscribe/unsubscribe or doing a navigation as in your first proposal.

Reply:
So maybe let's stick to this one then?

Reply:
IMHO having this subscription/unsubscription code in the fixture seems to be more stable given that the fixture is run before any line of the test code gets executed. So we should basically not badly interfere here, right?

Reply:
Yes in that specific context it's fine. We just need to be careful with those APIs.

Reply:
It doesn't work if you have a fixture that wants to subscribe to certain events, and then later you have a fixture which does subscribe/unsubscribe, since we'll unexpectedly end up in the unsubscribed state.